core/src/main/java/hudson/model/ManagementLink.java

```diff
@@ -131,4 +137,58 @@ public boolean getRequiresConfirmation() {
     public boolean getRequiresPOST() {
         return false;
     }
+
+    /**
+     * Name of the category for this management link. Exists so that plugins with core dependency pre-dating the version
+     * when this was introduced can define a category. Plugins with newer core dependency override {@link #getCategory()} instead.
+     *
+     * @return name of the desired category, one of the enum values of {@link Category}, e.g. {@code STATUS}.
+     * @since TODO
+     */
+    @Restricted(NoExternalUse.class) // TODO I don't think this works
```

Comment:
does it work or not? 😄

Reply:
It doesn't, but since it's a client-side bug in access-checker, it still makes sense to leave the annotation in. Perhaps this?

```suggestion
    @Restricted(NoExternalUse.class) // Doesn't work as of 2020-03 due to https://github.com/kohsuke/access-modifier/pull/20 but annotate anyway
```

Reply:
ah ok, yeah perfect